sdk/evm/src/ntt.ts

```diff
@@ -160,18 +162,24 @@ export class EvmNtt<N extends Network, C extends EvmChains>
 
   async *setWormholeTransceiverPeer(peer: ChainAddress<C>) {
     // TODO: we only have one right now, so just set the peer on that one
+    // in the future, these should(?) be keyed by attestation type
     yield* this.xcvrs[0]!.setPeer(peer);
   }
 
   async *transfer(
     sender: AccountAddress<C>,
     amount: bigint,
     destination: ChainAddress,
-    queue: boolean
+    queue: boolean,
+    relay?: boolean
   ): AsyncGenerator<EvmUnsignedTransaction<N, C>> {
-    const [_, totalPrice] = await this.quoteDeliveryPrice(destination.chain);
+    let totalPrice = 0n;
+    if (relay) {
+      [, totalPrice] = await this.quoteDeliveryPrice(destination.chain);
```

Comment:
`quoteDeliveryPrice` will return the wormhole fee (currently `0`) for manual transfers. I think we should call this even if `relay === false`. We will need to pass the `relay` flag to `quoteDeliveryPrice` and encode it